synfig-studio/src/gui/workarea.cpp

```diff
@@ -803,6 +812,22 @@ WorkArea::set_background_rendering(bool x)
 	queue_draw();
 }
 
+void
+WorkArea::set_rulers_visible(bool visible)
+{
+	hruler->set_visible(visible);
+	vruler->set_visible(visible);
+	menubutton_box->set_visible(visible);
+}
+
+void
+WorkArea::toggle_rulers()
```

Comment:
1) `toggle_rulers` can be safely replaced and removed:
```
set_rulers_visible(!get_show_rulers())
```
2) I suggest to rename `set_rulers_visible` to `set_show_rulers`, so we get `get_show_rulers`/`set_show_rulers`

Reply:
done